logstash-output-tcp.gemspec

```diff
@@ -25,7 +25,11 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'logstash-codec-json'
   s.add_runtime_dependency 'stud'
 
+  # we depend on bouncycastle's bcpkix-jdk15on being on the class-path
+  s.add_runtime_dependency 'jruby-openssl', '>= 0.12.2' # 0.12 supports TLSv1.3
```

Comment:
this requirement, can not be met in 7.x (till 7.17.3 is out) and thus is going to cause :red_circle: CI (for a few weeks)
initially thought of a hack of not forcing the requirement to keep CI :green_circle: (initially there were more failing targets) - which was discussed as not a good idea https://github.com/logstash-plugins/logstash-input-tcp/pull/198#discussion_r836798769= and got [reverted](https://github.com/logstash-plugins/logstash-output-tcp/pull/47/commits/dbdf66d67e1f4ff79313e56809bd84ba5b783356)